src/app/public/modules/autocomplete/autocomplete.component.scss

```diff
@@ -38,20 +37,32 @@
 .sky-autocomplete-actions {
   @include sky-border(dark, top);
 
+  .sky-autocomplete-action {
+    border: none;
+  }
+
   .sky-autocomplete-actions-no-results {
     display: inline-block;
     padding: 7px 13px;
   }
 }
 
-.sky-autocomplete-add {
+.sky-autocomplete-action-add {
   float: right;
 
   sky-icon {
     padding-right: $sky-theme-modern-space-xs;
   }
 }
 
+.sky-autocomplete-descendant-focus {
```

Comment:
I understand this change for modern theme. However, would like @Blackbaud-AdamFunderburk to verify this change for default theme. We are using modern theme mixins and design tokens here for something that is also in default theme and would cause the selected item state here to not match other dropdowns in default theme if we do this.

Reply:
I've updated this to no longer use modern mixins for default theme.